sr_demos/scripts/sr_demos/burn_in.py

```diff
@@ -159,30 +166,30 @@ def execute_command_check(hand_commander, joint_states_config, joint_states,
     args = parser.parse_args(rospy.myargv()[1:])
 
     if args.side == 'right':
-        joint_prefix = 'rh_'
+        joint_prefix_name = 'rh_'
     elif args.side == 'left':
-        joint_prefix = 'lh_'
+        joint_prefix_name = 'lh_'
     else:
-        joint_prefix = 'both'
+        joint_prefix_name = 'both'
 
-    if 'rh_' == joint_prefix:
+    if joint_prefix_name == 'rh_':
         hand_name = 'right_hand'
-    elif 'lh_' == joint_prefix:
+    elif joint_prefix_name == 'lh_':
         hand_name = 'left_hand'
     else:
         hand_name = 'two_hands'
 
-    hand_commander = SrHandCommander(name=hand_name)
+    hand_commander_ptr = SrHandCommander(name=hand_name)
 
     # Get joint states for burn in demo
     burn_in_config_filename = '/home/user/projects/shadow_robot/base/src/'\
```

Comment:
Since we use ROS can't we access this using rospkg so that it becomes less hardcoded and therefore more maintainable?

Reply:
Why not keep `hand_commander`

Reply:
Its used within a class later in the function and you can't have that according to the linter.

Reply:
ahhhh, you mean it complains that this might be re-defined?

Reply:
maybe `hand_commander_` would be less confusing?

Reply:
Yeah exactly!

Reply:
I think that would be more confusing as it'd make it look like a un-used variable, I think changing ptr to instance is much clearer

Reply:
all right